docs/.vitepress/config.ts

```diff
@@ -92,3 +98,34 @@ function getConfigSidebar() {
     }
   ]
 }
+
+function getHead(pageData: PageData): HeadConfig[] {
+
+  const site = 'https://vitepress.vuejs.org/';
+  const canonicalURL = `${site}${pageData.relativePath}`.replace(/.md$/, '.html');
+ 
+  return [
+        // Twitter
+        ['meta', { name: 'twitter:card', content: 'summary_large_image' }],
+        ['meta', { name: 'twitter:site', content: '@vuejs' }],
+        ['meta', { name: 'twitter:title', content: pageData.title }],
+        ['meta', { name: 'twitter:description', content: pageData.description || description }],
+
+        // Open Graph
+        ['meta', { property: 'og:type', content: 'website' }],
+        ['meta', { property: 'og:locale', content: lang }],
```

Comment:
`og:locale` uses [underscore](https://ogp.me/#optional) instead of the BCP 47 [hyphen](https://www.rfc-editor.org/rfc/rfc5646.html#page-4) in HTML [lang](https://html.spec.whatwg.org/multipage/dom.html#the-lang-and-xml:lang-attributes) to separate the subtags, so language tags like `en-US` should be converted to `en_US`.